tutorials/solution_strategies.ipynb

```diff
@@ -0,0 +1,224 @@
+{
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #74.            sd = subdomains[0]
</pre><p>Why not a loop over all subdomains? I guess from L114 that you know that the domain has only one fracture, and that you further know that this particular function will only be called on fracture subdomains. Neither of those will be clear to a reader of the tutorial, so I suggest adding some comments.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/pmgbergen/porepy/pull/1208/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='1'/>

Reply:
<p>You're right, that would have been better. Outdated now, though.</p>